src/dvr/dvr_db.c

```diff
@@ -3230,6 +3230,43 @@ dvr_entry_class_disp_extratext_get(void *o)
   return &prop_ptr;
 }
 
+static int
+dvr_entry_class_disp_extratext_set(void *o, const void *v)
+{
+  dvr_entry_t *de = (dvr_entry_t *)o;
+  const char *lang = idnode_lang(o);
+  const char *s = "";
+  int field = 0;
+  v = tvh_str_default(v, "UnknownExtraText");
+  if (de->de_subtitle){
+    s = lang_str_get(de->de_subtitle, lang);
```

Comment:
The lang_str_set() already returns 0 (no string change) or 1 (string changed), so you can do 'return lang_str_set(de->de_substitle, v, lang);' here.